index.js

```diff
@@ -1095,10 +1098,15 @@ Multi.prototype.exec = function (callback) {
                     args[args.length - 1](null, reply);
                 }
             }
+        } else if (err) {
+            err.code = 'EXECABORT';
+            err.errors = errors;
         }
 
         if (callback) {
-            callback(null, replies);
+            callback(err, replies);
```

Comment:
On newer redis' is it possible to have a multi response that contains errors and replies, I thought the transaction is aborted -- so you wouldn't get anything.

Is there a scenario where we would get both errors and replies?